### DISCUSSION TITLE
Tell us how you are using Bionic GPT.

### DISCUSSION THREAD

**Discussion by ianpurton** [bionic-gpt/bionic-gpt#244] [Show and tell]
We love to here how and where you are installing Bionic and any use cases it was able to help you with.

**Comment by tbendien** [comment]
Here are some use cases:

1. Teaching school students how to learn prompt engineering on a private generative AI system where chat history can be audited. 
2. Creating content for marketing agencies privately as they don't want to share confidential information with any public AI service prior to the time that sensitive company information is released to the public. For example, creating a PR that has company earnings mentioned.
3. Creating SQL queries. This needs to stay private as sensitive company information such as column names and other things will be included in the SQL.
4. Finding specialized Linux commands.
5. Creating steps to troubleshoot IT helpdesk related issues - this data needs to be kept private to the organization.
6. Being able to run analytics on prompt and response history to develop insights into user behavior and look for common themes/questions that could highlight the need for employee training or fixing a business issue that appears on a regular basis in the prompts.
